src/Exception/LogicException.php

```diff
@@ -10,13 +10,23 @@
 class LogicException extends \LogicException implements MatejExceptionInterface
 {
     public static function forInconsistentUserId(UserAwareInterface $mainCommand, UserAwareInterface $additionalCommand)
+    {
+        return self::forInconsistentRawUserId(
+            $mainCommand->getUserId(),
+            (new \ReflectionClass($mainCommand))->getShortName() . ' command',
+                $additionalCommand->getUserId(),
+            (new \ReflectionClass($additionalCommand))->getShortName() . ' command'
+        );
+    }
+
+    public static function forInconsistentRawUserId($mainUserId, $mainSubject, $additionalUserId, $additionalSubject)
```

Comment:
Maybe in the two cases where this method is used directly, it could be replaced with own specific method with less generic name (as the mainSubject and additionalSubject values are always the same when it is called directly).